lib/workos/webhooks.rb

```diff
@@ -55,8 +55,25 @@ def construct_event(
         WorkOS::Webhook.new(payload)
       end
 
-      private
-
+      # Verifies WorkOS-Signature header from request
+      # rubocop:disable Layout/LineLength
+      #
+      # @param [String] payload The payload from the webhook sent by WorkOS. This is the RAW_POST_DATA of the request.
+      # @param [String] sig_header The signature from the webhook sent by WorkOS.
+      # @param [String] secret The webhook secret from the WorkOS dashboard.
+      # @param [Integer] tolerance The time tolerance in seconds for the webhook.
+      #
+      # @example
+      #   WorkOS::Webhooks.verify_header(
+      #     payload: "{"id": "wh_123","data":{"id":"directory_user_01FAEAJCR3ZBZ30D8BD1924TVG","state":"active","emails":[{"type":"work","value":"blair@foo-corp.com","primary":true}],"idp_id":"00u1e8mutl6wlH3lL4x7","object":"directory_user","username":"blair@foo-corp.com","last_name":"Lunceford","first_name":"Blair","directory_id":"directory_01F9M7F68PZP8QXP8G7X5QRHS7","raw_attributes":{"name":{"givenName":"Blair","familyName":"Lunceford","middleName":"Elizabeth","honorificPrefix":"Ms."},"title":"Developer Success Engineer","active":true,"emails":[{"type":"work","value":"blair@foo-corp.com","primary":true}],"groups":[],"locale":"en-US","schemas":["urn:ietf:params:scim:schemas:core:2.0:User","urn:ietf:params:scim:schemas:extension:enterprise:2.0:User"],"userName":"blair@foo-corp.com","addresses":[{"region":"CO","primary":true,"locality":"Steamboat Springs","postalCode":"80487"}],"externalId":"00u1e8mutl6wlH3lL4x7","displayName":"Blair Lunceford","urn:ietf:params:scim:schemas:extension:enterprise:2.0:User":{"manager":{"value":"2","displayName":"Kathleen Chung"},"division":"Engineering","department":"Customer Success"}}},"event":"dsync.user.created"}",
```

Comment:
Are you sure you want to put your name in the examples here? It's not exactly a secret - the GitHub profile is there, but if it's in the code, it may well find a way to spread further 🤷‍♂️

Reply:
Yeah I think I was just lazy, and pulled a webhook from my testing 🤦‍♀️ Let me see if I can easily change this example payload.

Reply:
Though it is just my name and where I live, which is also on my GitHub and LinkedIn profiles - the email is not real

Reply:
Yeah, I doubt it really makes all that much of a difference, but I generally feel like the less PII, the better ... even if it's not 100% identifiable 🤷‍♂️. I suppose we could also spin as setting a good example of how careful we are when handling customer PII data...